### ISSUE TITLE
pressKeys event does not contain code

### ISSUE THREAD

**Issue by varunjayaraman** [theintern/intern#835]
## Expected behavior

It appears that the `pressKeys` functionality is not emitting a `KeyboardEvent` with the correct `code` properties. For instance, when invoking `.pressKeys('SPACE')`, I would expect the `event` that gets passed to my listener on `keydown` to have the `code` `Space` or the `keyCode` as `32`.

## Current behavior

Currently, `event.keyCode` is undefined and it seems `code` is `S` (instead of `Space`)

## Possible solution
My guess is the problem stems from here: https://github.com/theintern/leadfoot/blob/master/src/Session.ts#L2216-L2246

I'm not 100% sure but shouldnt the `code` or `keyCode` properties get set here?

## Steps to reproduce (for bugs)

I've created a simple html file:

```html
<html>

<head>

</head>

<body>

  <div id="scrollDepth"></div>
  <script>
    document.addEventListener('keydown', function (ev) {
      alert(`Keycode: ${ev.code}`);
    });
  </script>
</body>

</html>
```

Then in my intern test:

```javascript
    'Should track keydown spacebar': function () {
      return this.remote.get(paths.testKeydown)
        .setWindowSize(800, 3000)
        .sleep(5000)
        .pressKeys('SPACE')
        .sleep(5000)
        .end()
    },
```

When the alert pops up, it shows `Keycode: ` (so ev.code mus be undefined). What's interesting is that it pops the alert up 5 times (I wonder if it interprets `SPACE` as 5 different keys?). I've tried passing in `SPACE` as an array, like `.pressKeys(['SPACE'])` with the same result.

1. …
2. …
3. …

## Environment

Intern version: 3.4.3
Node version: 8.0
NPM version: 5.0
Browser version: chrome (im assuming latest, we're just passing `{ browserName: 'chrome' }` to the environments array in the config

**Comment by jason0x43** [comment]
That code shouldn't actually be coming into play for Chrome; it's intended as a fallback for browsers that don't support native events via Webdriver. We'll look into that.

`pressKeys` takes a string of characters that should be pressed. To press the space key, send a space, like `pressKeys(' ')` or `pressKeys(keys.SPACE)` (if you've imported the `keys` module).

**Comment by varunjayaraman** [comment]
I see. the documentation is pretty sparse on this front. Any way you could add actual usage examples to the docs / how to import keys such that it's available? 

As for the solution you suggested, i changed it to `.pressKeys(' ')` and my alert still logs out an undefined event code with:

```javascript
alert(`Event Code: ${ event.code }`)
```

**Comment by jason0x43** [comment]
Yes, we can definitely add some examples (I added [an issue](https://github.com/theintern/leadfoot/issues/129) to track this in the Leadfoot repo).

The current docs for `pressKeys`, as well as the other WebDriver functionality, are in the Leadfoot docs (Leadfoot is Intern's WebDriver interface). For `pressKeys`, see https://theintern.io/docs.html#Leadfoot/2/api/Command/presskeys.

**Comment by ajc24** [comment]
Can someone please give me an example of how to import the `keys` module?

**Comment by jason0x43** [comment]
@ajc24 See https://github.com/theintern/leadfoot/issues/129#issuecomment-348997790. Please try to keep comments relevant to the issue.
